src/photErrorModel.py

```diff
@@ -38,8 +38,9 @@ def __init__(self, pars={}):
         if (not isinstance(pars, dict)):
             raise TypeError("Error parameters passed to photometric error model not in dictionary structure")  
         self.setModelPars(pars)
-    
-    
+        random.seed(pars['randSeed'])
```

Comment:
It seems like this should really be done in setModelPars, rather than here in the init, can you move this line to the two setModelPars functions?

Reply:
I do agree and it was the first thing in my mind. I've tried doing it but it did not work.

Reply:
Thanks @sschmidt23 for your edit suggestions. Now users can either do

``` python
import photErrorModel as pem
pars={'randSeed': 2019, ...}
errmodel = pem.LSSTErrorModel(pars)
```
or
``` python
import photErrorModel as pem
pars={...}
errmodel = pem.LSSTErrorModel(pars)
errmodel.setRandomSeed(2019)
```

This works for FractionalErrorModel as well.